docs/requirements.txt

```diff
@@ -1,5 +1,4 @@
 autodoc-traits
-myst_parser[sphinx]
-sphinx<3
```

Comment:
myst_parser requires sphinx explicitly now, and the version needed is above sphinx 3 for autodoc traits at least.